docker/scripts/common.sh

```diff
@@ -63,6 +63,26 @@ e2e_test_fill() {
     echo Fill tx hash: $e2e_test_l2_txhash
 }
 
+e2e_test_relayer() {
+    local root=$(get_root_dir)
+    local l1_rpc=$1
+    local l2_rpc=$2
+    local network_config=$3
+    local privkey=$4
+    local txhash=$5
+    local relayer=${root}/relayer/relayer-node18-linux-x64
+    
+    echo Starting relayer...
+    timeout 5m bash -c "until ${relayer} --l1-rpc-url '$l1_rpc' \
```

Comment:
I think we should omit the timeout and let CircleCI's timeout handle that. In some cases even 5min may not be enough so let's just defer that to CircleCi.

Reply:
Sure we can do that.
I did it because you said the opposite in the issue: https://github.com/beamer-bridge/beamer/issues/1805#issuecomment-1547705673

Reply:
Yeah, I think I was not aware just how long this stuff can take when I wrote that comment. We may even have to increase the CircleCI timeout. :disappointed:

Reply:
We just talked in person and decided to merge and see if we run into issues with the timeout.